### DISCUSSION TITLE
sidekiq failures

### DISCUSSION THREAD

**Discussion by yossiedri** [sidekiq/sidekiq#5879] [Q&A]
Hi Mike
is there a way to create sidekiq failures entry programatically?

**Comment by mperham** [comment]
There is no such thing as a “failure” entry in Sidekiq. You’ll need to be more specific. Are you talking about an entry in the Retry set? Tell me more about what you are trying to do.

**Reply by yossiedri** [reply]
Hi Mike
when a job fails i can see in in the Failed tab in the web interface with the stack trace and all,
this entry is created since an exception was raised in the execution,
my question is if i can create that entry by code since i dont want to raise an exception but use the rescue block to add it to the Failed tab.
the reason i want to do it is that the exception terminate the process but do not release the connection to the DB and i want to avoid it.

**Comment by mperham** [comment]
The Failed tab is not part of Sidekiq. It is a 3rd party extension. Remove it and use Retries.

**Reply by yossiedri** [reply]
Mike i removed the "Sidekiq-failures" gem and now i see the failed job count,
is there a way to reset that count and see the failed job stack trace in sidekiq ent?

**Reply by mperham** [reply]
Failed jobs go into the Retries tab where you can see the backtrace. Please review the Error Handling wiki page to learn how Sidekiq's built-in error handling works.

**Reply by yossiedri** [reply]
is there a way to clear failed count in the sidekiq ent web?

**Reply by mperham** [reply]
`Sidekiq::Stats.new.reset(:failed)` but I'm not sure why you'd ever want to do this.
